src/components/dialogs/limits/limitsChart.tsx

```diff
@@ -0,0 +1,239 @@
+                const difference = item.value ? item.value - previousSum : undefined;
+
+                const color =
+                    isIst || !item.tempo || (item.tempo && item.value && maxValueIst >= item.value)
+                        ? colorIST
+                        : colors?.[colorIndex] ?? colors[colors.length - 1];
+
+                const incoherent = isIncoherent(maxValueIst, item, index > 0 ? data[index - 1] : undefined);
+
+                if (item.value && item.value >= maxValueIst) {
+                    previousSum = item.value;
+                }
+                if (item.value && item.value > maxValueIst) {
+                    colorIndex++;
+                }
+
+                const updatedSeries: BarSeriesType[] =
+                    (item.tempo && !item.value) || (item.value && item.value >= maxValueIst)
+                        ? [
+                              ...acc.series,
+                              {
+                                  type: 'bar',
+                                  label:
+                                      isIst || item.value === maxValueIst
+                                          ? intl.formatMessage({ id: 'unlimited' })
+                                          : formatTempo(item.tempo),
+                                  data: [difference ?? maxValue * 0.15],
+                                  color: item.value ? color : colorForbidden,
+                                  stack: 'total',
+                              },
+                          ]
+                        : [...acc.series];
+                const updatedTicks: Ticks[] = [
+                    ...acc.ticks,
+                    { position: item.value, label: item.label, incoherent: incoherent },
+                ];
+
+                if (
+                    index === data.length - 1 &&
+                    updatedTicks[updatedTicks.length - 1].position &&
+                    !noValueThresholdFound
+                ) {
+                    updatedSeries.push({
+                        type: 'bar',
+                        label: intl.formatMessage({ id: 'forbidden' }),
+                        data: [(updatedTicks?.[updatedTicks?.length - 1]?.position ?? 0.0) * 0.15],
+                        color: colorForbidden,
+                        stack: 'total',
+                    });
+                }
+                return {
+                    series: updatedSeries,
+                    ticks: updatedTicks,
+                };
+            },
+            { series: [], ticks: [] }
+        );
+    }, [currentLimits, intl, formatTempo, isIncoherent]);
+
+    const config = {
+        id: 'topAxis',
+        valueFormatter: (value: number, context: AxisValueFormatterContext) =>
+            ticks.find((item: Ticks) => item.position === value)?.label,
+    };
+
+    return (
+        <BarChart
+            margin={{ left: 0, right: 0 }}
+            height={140}
```

Comment:
Would reduce the big top margin : 
```suggestion
            margin={{ left: 0, right: 0, top: 20 }}
            height={110}
```